Documentation/ReleaseNotes.md

```diff
@@ -114,4 +118,10 @@ The stored initial world pose `worldPoseOnManipulationStart` data type has been
 
 A new property `AudioClip` has been added to the IMixedRealityDictationSystem interface. The `AudioClip` property enables access to the audio clip associated with the current dictation session. Users must implement the property in their scripts implementing the interface.
 
+**Addition of Motion Controller to Input Simulation Service**
+
+Motion Controller simulation is now offered in editor play mode along side the existing hand simulation. To enable this change, extensive naming changes have been made to input simulation related files, with `InputSimulationService.cs` and `MixedRealityInputSimulationProfile.cs` getting the most significant changes. The logic and behavior of relevant code largely remain the same, and the majority of naming changes are related to replacing reference to "hand" to the more generic term "controller" (e.g. from `DefaultHandSimulationMode` to `DefaultControllerSimulationMode`). The existing fields, properties and functions are marked obsolete. Besides renaming, the return type of certain functions are changed to match the name/behavior change (e.g. `GetControllerDevice` based on the original `GetHandDevice` now returns `BaseController` instead of `SimulatedHand`).
```

Comment:
For functions/properties/classes that were renamed rather than obsoleted, it would be helpful to have a table here that lists both the old and the new.

i.e. it's minimally helpful to see:

```
To enable this change, extensive naming changes have been made to input simulation related files, with `InputSimulationService.cs` and `MixedRealityInputSimulationProfile.cs` getting the most significant changes. The logic and behavior of relevant code largely remain the same, and the majority of naming changes are related to replacing reference to "hand" to the more generic term "controller"
```

It's more helpful to see something like:

| Old  | New  |
|---|---|
| DefaultHandSimulationMode  | DefaultControllerSimulationMode |
| BaseInputSimulationService::UpdateHandDevice  |  BaseInputSimulationService::UpdateControllerDevice  |

... etc

Reply:
Basically, similar to (but obviously smaller scope than):

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/HTKToMRTKPortingGuide.html

Reply:
Sorry for the imprecise wording - the changes are actually marking things obsolete rather than renaming. The obsolete messages include what the developer should now be using.

Reply:
Chatted offline on this - this was a bit hard to see because of review deltas but we actually kept the old fields. i.e. my reading here was wrong (the new names and the old names were fairly far apart in the file).